src/bin/client.rs

```diff
@@ -0,0 +1,56 @@
+use std::io::{self, Write};
+use std::time::Duration;
+use std::thread;
+use futures_util::{future, pin_mut, StreamExt};
+use tokio::io::{AsyncReadExt, AsyncWriteExt};
+use tokio_tungstenite::{connect_async, tungstenite::protocol::Message};
+use clap::Parser;
+
+#[derive(Parser, Debug)]
+#[clap(version, about)]
+struct Args {
+    #[clap(short, long, default_value = "ws://localhost:8000/opcode")]
+    connect_addr: String,
+}
+
+#[tokio::main]
+async fn main() {
+    let args = Args::parse();
+
+    let url = url::Url::parse(&args.connect_addr).unwrap();
+    let (stdin_tx, stdin_rx) = futures_channel::mpsc::unbounded();
```

Comment:
```suggestion
   futures_channel::mpsc::unbounded();
```
`tokio::sync::mpsc::unbounded_channel()`?